.github/workflows/ci.yml

```diff
@@ -119,6 +125,13 @@ jobs:
     - name: Restore $GITHUB_ENV
       run: cat .github_env >> $GITHUB_ENV
 
+    - name: Generate complete Docker image tags
+      run: |
+        IMAGE=${{ env.docker_repository_base }}-${{ matrix.model-name }}
+        TAGS=${{ env.tags }}
+        NEW_TAGS="$(echo $TAGS | tr "," "\n" | awk -v image="$IMAGE" '{print image ":" $1}' | paste -sd "," -)"
```

Comment:
This line may be the source of the issue `invalid tag "docker.synapse.org/syn22277123/phi-annotator-huggingface-dslim/bert-large-NER::pr-24": invalid reference format`. Not sure why a second `:` is inserted after the first one.